PoGo.NecroBot.CLI/Settings.cs

```diff
@@ -100,7 +100,6 @@ public static GlobalSettings Load(string path)
             {
                 settings = new GlobalSettings();
             }
-            Logger.Write($"nigger {fullPath} {ConfigPath} {ProfilePath}");
```

Comment:
I was just curious what it was.  is it the actual N word, or like an
abbreviation or something?

On Tue, Jul 26, 2016 at 4:38 PM, 5andr0 <notifications@github.com> wrote:

> In PoGo.NecroBot.CLI/Settings.cs
> <https://github.com/NecronomiconCoding/NecroBot/pull/667#discussion_r72331946>
> :
>
> > @@ -100,7 +100,6 @@ public static GlobalSettings Load(string path)
> >              {
> >                  settings = new GlobalSettings();
> >              }
> > -            Logger.Write($"nigger {fullPath} {ConfigPath} {ProfilePath}");
>
> That was a debug output. Sorry for the n word, not racist ;) I've been
> using this for ages to mark my debug outputs
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/NecronomiconCoding/NecroBot/pull/667/files/c2774808216d96d8db60011e282ef02079c11fe1..59dce97b0209621a2ce2f477ee7650e4d5b46c61#r72331946>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATo2QVzglb5wIza85ZWGWZKE6dm6UCnKks5qZnAzgaJpZM4JVdmJ>
> .
>